Dockerfile

```diff
@@ -66,7 +66,7 @@ RUN if [ "$cq_version" = "release" ] ; \
 RUN if [ "$cq_version" = "master" ] ; \
     then conda install jupyter -y --quiet ; \
     conda clean -afy ; \
-    conda install -c cadquery -c conda-forge cadquery=master ; \
+    conda install -c cadquery -c conda-forge cadquery="$cq_version" ; \
```

Comment:
this is a nice approach to use the varible in the install command, we might be able to reduce the dockerfile size using this method. I can look into that separately. Perhaps we change the accepted arguments to "2" and "master" and then users can get what they want directly

Reply:
> Perhaps we change the accepted arguments to "2" and "master" and then users can get what they want directly

Didn't get that I'm afraid.